lib/rb_shift/project.rb

```diff
@@ -19,39 +20,35 @@ def self.open(name, client)
       Project.new(name, client)
     end
 
-    def pods(update = false)
-      @_pods = init_objects(Pod) if update || @_pods.nil?
-      @_pods
+    def pods(fetch: false, name: nil)
+      get_instance_variable name: :pod, type: Pod, reload: fetch, filter: name
```

Comment:
```reload``` is even better name ... so i suggest we use that instead of ```fetch```... also I hear that consistency is a good thing.